### PR TITLE
refactor(VListItem): implement grid, normalize usage w/ v-card-item

### PR DESCRIPTION
**Pull request by johnleider**
let's see if Kael notices me

## Description
update to use grid, removed sub-components (`v-list-item-avatar`, `v-list-item-icon`)

## Motivation and Context
update with style changes applied to `v-card-item`

## How Has This Been Tested?
todo

## Markup:
I'd also suggest building and looking at the docs locally.

<details>

```vue
<template>
  <v-card
    class="mx-auto"
    width="300"
  >
    <v-list v-model:opened="open">
      <v-list-item prepend-icon="mdi-home" title="Home" color="primary" />

      <v-list-group
        title="Users"
        subgroup
      >
        <template #activator="{ props }">
          <v-list-item v-bind="props" color="primary" />
        </template>

        <v-list-group title="Admin" prepend-icon="mdi-vuetify">
          <template #activator="{ props }">
            <v-list-item v-bind="props" />
          </template>

          <v-list-item
            v-for="([title, icon], i) in admins"
            :key="i"
            :title="title"
            :prepend-icon="icon"
            :value="title"
          />
        </v-list-group>

        <v-list-group prepend-icon="mdi-account" value="Actions" title="Actions">
          <template #activator="{ props }">
            <v-list-item v-bind="props" />
          </template>

          <v-list-item
            v-for="([title, icon], i) in cruds"
            :key="i"
            :value="title"
            :title="title"
            :prepend-icon="icon"
          />
        </v-list-group>
      </v-list-group>
    </v-list>

    <pre>{{ JSON.stringify(open, null, 2) }}</pre>
  </v-card>
</template>

<script>
  export default {
    data: () => ({
      open: ['Users'],
      admins: [
        ['Management', 'mdi-account-multiple-outline'],
        ['Settings', 'mdi-cog-outline'],
      ],
      cruds: [
        ['Create', 'mdi-plus-outline'],
        ['Read', 'mdi-file-outline'],
        ['Update', 'mdi-update'],
        ['Delete', 'mdi-delete'],
      ],
    }),
  }
</script>
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VList/VListChildren.tsx [hunk 1]
```diff
@@ -49,9 +49,12 @@ export const VListChildren = genericComponent<new <T extends InternalListItem>()
         title: slots.title ? (slotProps: any) => slots.title?.({ ...slotProps, item }) : undefined,
       }
 
+      const [listGroupProps, _1] = filterListGroupProps(itemProps as any)
+
       return children ? (
         <VListGroup
           value={ itemProps?.value }
```
Comment (KaelWD) [inline packages/vuetify/src/components/VList/VListChildren.tsx:56]: Is this already included in listGroupProps?

Comment (johnleider) [reply packages/vuetify/src/components/VList/VListChildren.tsx:56]: Yes but everything blows up when I remove it.

**Comment by zorn-v** [comment]
Did not investigate so deep, but how to convert this
изображение
to this (as was before beta.6) ?
изображение
Seems there even "prepend" slot does not work

I have left beta.5 for now

```html
    <v-list density="compact" nav v-else>
      <v-list-item
        v-for="project in projects"
        :key="project._id"
        :to="{ name: 'project', params: { id: project._id } }"
        :title="project.title || project.name"
      >
        <template #prepend>
          <v-list-item-icon icon="mdi-monitor-edit" class="mr-3" />
        </template>
        <v-list-item-media end>
          <v-chip color="accent" size="small">{{ project.items?.length }}</v-chip>
        </v-list-item-media>
      </v-list-item>
    </v-list>
```

**Comment by nekosaur** [comment]
https://codepen.io/nekosaur/pen/dyePYjN?editors=1010

**Comment by zorn-v** [comment]
Thank you

**Pull request merged**